index.bs

```diff
@@ -952,11 +952,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>[WAA]</dfn>
 :: A cryptographic entity, existing in hardware or software, that can [=registration|register=] a user with a given [=[RP]=]
     and later [=Authentication Assertion|assert possession=] of the registered [=public key credential=], and optionally
-    [=user verification|verify the user=], when requested by the [=[RP]=]. [=Authenticators=] can report information
```

Comment:
Sorry for this extreme nitpicking, but I disagree with deleting this comma. Some authenticators require user verification even when not requested by the RP; this can even be a preference configured by the user in some cases.

Reply:
I would say that the better solution would be to replace 
> verify the user, when requested by the RP" 

with 

> " verify the user to the RP"

the current format with the comma makes an already run-on sentence have improper syntax.

Reply:
Ok, that too seems fine to me.